### PR TITLE
Add Changelog file (#278) 

### PR DESCRIPTION
**Pull request by ionut-arm**
Signed-off-by: Ionut Mihalcea <ionut.mihalcea@arm.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+- Investigate and define the work required for SPIFFE-based client identity management [\#232](https://github.com/parallaxsecond/parsec/issues/232)
+- Remove PKCS11 single thread lock \(\#264\) [\#277](https://github.com/parallaxsecond/parsec/pull/277) ([ionut-arm](https://github.com/ionut-arm))
+
+**Fixed bugs:**
+
+- PKCS11 multi-threading [\#264](https://github.com/parallaxsecond/parsec/issues/264)
+
+**Closed issues:**
+
+- Investigate and define the work required for compatibility with Arm Firmware Framework for Armv8-A \(FF-A\) [\#247](https://github.com/parallaxsecond/parsec/issues/247)
```
Comment (hug-dev) [inline CHANGELOG.md:18]: Can we keep those "Unreleased" changes for the next release?

**Review by hug-dev — COMMENTED** [review]
Overall it looks really good and I am impressed with the quality of the file produced! Just one comment about unreleased items.

File: CHANGELOG.md (hunk 1 above)
Comment (ionut-arm) [reply CHANGELOG.md:18]: I can update it for that! it just needs a flag attached to the command

Comment (hug-dev) [reply CHANGELOG.md:18]: oh nice! Yes then I would be in favour of doing that for this PR and the other ones as well.

**Review by hug-dev — APPROVED** [review]

**Pull request merged**